support/pgbouncer-build

```diff
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-PGBOUNCER_VERSION=${PGBOUNCER_VERSION-1.17.0}
+PGBOUNCER_VERSION=${PGBOUNCER_VERSION-1.21.0}
```

Comment:
Is a version upgrade needed to work around the process signal issue? Can we separate that out from the prepared statements support?